udata/frontend/helpers.py

```diff
@@ -23,7 +25,11 @@
 
 @front.app_template_global(name='static')
 def static_global(filename):
-    return url_for('static', filename=filename)
+    if current_app.config['DEBUG']:
+        burst = time()
+    else:
+        burst = pkg_resources.get_distribution('udata').version
+    return url_for('static', filename=filename, _=burst)
```

Comment:
I'm curious about `_=burst`, isn't it `_anchor=burst`? That parameter doesn't look to exist in the [Flask's documentation](http://flask.pocoo.org/docs/0.10/api/#flask.url_for).